### PR TITLE
Added support for Vigintos Modulator & Ampiflier

### PR DESCRIPTION
**Pull request by jozefrebjak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
@jozefrebjak Looks like you could have used yaml for this any reason not to?

File: mibs/viginitos/VEL-MIB [hunk 1]
```diff
@@ -0,0 +1,41 @@
+VEL-MIB DEFINITIONS ::= BEGIN
```
Comment (murrant) [inline mibs/viginitos/VEL-MIB]: The MIB directory should match the os name. (you have an extra i here)

Comment (jozefrebjak) [reply mibs/viginitos/VEL-MIB]: it was mistake, I changed it

Comment (jozefrebjak) [reply mibs/viginitos/VEL-MIB]: thank you :)

**Comment by jozefrebjak** [comment]
There is how it is looks like in a web, its something like host controller for DVB modulator and amplifier

snimka obrazovky 2018-12-21 o 22 33 00
snimka obrazovky 2018-12-21 o 22 32 51

**Comment by murrant** [comment]
I'm saying according to the MIB it doesn't guarantee they will always be at those OIDs (on different devices).  It is just a list of values.

It might be more reliable to use host2DevParUnit to determine what type of sensor it is.

**Comment by murrant** [comment]
@jozefrebjak how are you coming along with this?

**Comment by jozefrebjak** [comment]
@murrant  so, i deleted all php discovery and replaced them with yaml. I used your advice to do this.

**Pull request merged**